_docs/policies/testing.md

```diff
@@ -0,0 +1,121 @@
+# `status-go` Test Policy
+
+## Purpose
+
+This policy defines the requirements and guidelines for testing within the `status-go` repository. It aims to ensure all new functionality is introduced with robust, reliable, and repeatable tests 
+while preventing flaky tests from being merged into the codebase. This aligns with Policy Zero’s principles of transparency, collaboration, and quality assurance in the development process.
+
+## Submitting and Maintaining the Test Policy
+
+- Any individual MAY propose amendments to this policy.
+- Proposed changes MUST be submitted as a pull request (PR) to the `_docs/policies` directory in the `status-go` repository.
+- The PR MUST include a brief justification for the amendment, answering: "Why is this change needed?"
+- The PR MUST follow the [Review and Approval Process](./introduce-policy.md#review-and-approval-process) outlined in Policy Zero.
+
+## Creating Tests
+
+- All new functionality MUST be introduced with tests that:
+  - Prove that the functionality performs as described
+  - Can be falsified
+  - Are resistant to fuzzing
+- All new `functional tests` MUST BE validated via a minimum of 1000 test runs.
```

Comment:
I don't know functional tests that well, but `integration tests` are the usual culprits of flakiness, so either this is a swap or we need to add `integration` there too